modules/tinymce/src/themes/silver/test/ts/browser/editor/core/LineHeightTest.ts

```diff
@@ -0,0 +1,137 @@
+import { Assertions, Chain, GeneralSteps, Log, UiFinder } from '@ephox/agar';
+import { UnitTest } from '@ephox/bedrock-client';
+import { Arr, Optional } from '@ephox/katamari';
+import { Editor as McEditor, TinyApis, TinyUi } from '@ephox/mcagar';
+import { Attribute, SugarElement } from '@ephox/sugar';
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+const settings = {
```

Comment:
This way of doing this is pretty inconsistent with the rest of the code, and makes it less readable (ie the start of the test is halfway down). Can you please either move these within the `UnitTest` block, or move them to a separate utils module.